packages/connector-aliyun-dm/src/constant.ts

```diff
@@ -0,0 +1,29 @@
+import path from 'path';
+
+import { ConnectorType, ConnectorMetadata } from '@logto/connector-types';
+import { getFileContents } from '@logto/shared';
+
+// eslint-disable-next-line unicorn/prefer-module
+const currentPath = __dirname;
```

Comment:
why disable?

Reply:
`__dirname` is not allowed.
<img width="723" alt="image" src="https://user-images.githubusercontent.com/15182327/165207097-2760229c-4d36-4b7f-991c-1ab04645256f.png">

Tried to fix this as following, but new problem came out, and can not fix this according to solutions posted on StackOverflow.
<img width="659" alt="image" src="https://user-images.githubusercontent.com/15182327/165209731-9fb8c601-0746-4840-8df0-d138d77c4a53.png">
<img width="1468" alt="image" src="https://user-images.githubusercontent.com/15182327/165209778-eec144cc-a7b7-464f-9066-4a5adac2492d.png">

Reply:
ok cool